api/tests/unit/domain/models/SkillReview_test.js

```diff
@@ -68,6 +68,123 @@ describe('Unit | Domain | Models | SkillReview', () => {
+        // Given
+        const targetedSkills = [skillLevel1, skillLevel2];
+        const validatedSkills = [skillLevel1];
+        const failedSkills = [skillLevel2];
+
+        // When
+        const skillReview = new SkillReview({ targetedSkills, validatedSkills, failedSkills });
+
+        // Then
+        expect(skillReview.profileCompletionRate).to.eq(1);
+      });
+
+      it('should returns a profileCompletionRate of 1 when all skills are failed', () => {
+        // Given
+        const targetedSkills = [skillLevel1, skillLevel2];
+        const validatedSkills = [];
+        const failedSkills = [skillLevel1, skillLevel2];
+
+        // When
+        const skillReview = new SkillReview({ targetedSkills, validatedSkills, failedSkills });
+
+        // Then
+        expect(skillReview.profileCompletionRate).to.eq(1);
+      });
+
+      it('should returns a profileCompletionRate of 1 when all skills are validated', () => {
+        // Given
+        const targetedSkills = [skillLevel1, skillLevel2];
+        const validatedSkills = [skillLevel1, skillLevel2];
+        const failedSkills = [];
+
+        // When
+        const skillReview = new SkillReview({ targetedSkills, validatedSkills, failedSkills });
+
+        // Then
+        expect(skillReview.profileCompletionRate).to.eq(1);
+      });
+
+      it('should returns a profileCompletionRate of 1 when some skills were not suited to be evaluated (too hard, no question)', () => {
+        // Given
+        const targetedSkills = [skillLevel1, skillLevel2];
+        const validatedSkills = [skillLevel1];
+        const failedSkills = [];
+        const unratableSkills = [skillLevel2];
+
+        // When
+        const skillReview = new SkillReview({ targetedSkills, validatedSkills, failedSkills, unratableSkills });
+
+        // Then
+        expect(skillReview.profileCompletionRate).to.eq(1);
+      });
+
+    });
+
+    context('with the skillProfile contains no skill', () => {
+      it('should returns a progression rate of 0', () => {
+        // Given
+        const targetedSkills = [];
+        const validatedSkills = [skillLevel1];
+        const failedSkills = [];
+
+        // When
+        const skillReview = new SkillReview({ targetedSkills, validatedSkills, failedSkills });
+
+        // Then
+        expect(skillReview.profileCompletionRate).to.eq(0);
+      });
+    });
+
+    context('with the skillProfile contains extra skills', () => {
+      it('should returns a progression rate of 0', () => {
+        // Given
+        const targetedSkills = [skillLevel1];
+        const validatedSkills = [skillLevel1, skillLevel2];
+        const failedSkills = [];
+
+        // When
+        const skillReview = new SkillReview({ targetedSkills, validatedSkills, failedSkills });
+
+        // Then
+        expect(skillReview.profileCompletionRate).to.eq(1);
+      });
+
+      it('should mark the completion at 100%', () => {
```

Comment:
should mark the completion at 1 (equal to 100%) ?

Reply:
👍